packages/contracts-bedrock/snapshots/storageLayout/L2CrossDomainMessenger.json

```diff
@@ -113,16 +106,30 @@
   },
   {
     "bytes": "20",
-    "label": "otherMessenger",
+    "label": "spacer_207_0_20",
     "offset": 0,
     "slot": "207",
-    "type": "contract CrossDomainMessenger"
+    "type": "address"
   },
   {
-    "bytes": "1376",
-    "label": "__gap",
+    "bytes": "20",
+    "label": "xDomainMsgSender",
     "offset": 0,
     "slot": "208",
-    "type": "uint256[43]"
+    "type": "address"
+  },
+  {
+    "bytes": "12",
+    "label": "spacer_208_20_12",
+    "offset": 20,
+    "slot": "208",
+    "type": "bytes12"
+  },
+  {
+    "bytes": "1312",
+    "label": "__gap",
+    "offset": 0,
+    "slot": "209",
+    "type": "uint256[41]"
```

Comment:
Two slots were removed from the gap on the left hand side.
1. to make space for the `xDomainMsgSender`
2. makes spaces for the `superchainConfig` in the L1 CrossDomainMessenger (with no negative effect on this contract).